README.md

```diff
@@ -1,3 +1,5 @@
+<img align="right" src="https://cdn.rawgit.com/slamdata/matryoshka/master/resources/matryoshka.svg" height="200px" style="padding-left: 20px"/>
```

Comment:
I think here you just want `src="resources/matryoshka.svg"`. IIRC, GitHub has fixed things so that works correctly. But maybe that only works if you use the `![logo](resources/matryoshka.svg)` syntax.

Regardless, that’s how it _should_ be, so it works independently of GitHub as well.